nxcompshad/Shadow.cpp

```diff
@@ -36,6 +36,13 @@
 #include "Poller.h"
 #include "Manager.h"
 
+/* On GNU/Hurd, we don't have asm-generic/errno.h (where
+ * EBADFD is defined on Linux systems).
+ */
+#ifndef EBADFD
+# define EBADFD 77
```

Comment:
I don't have a GNU/Hurd system handy, is there really no such macro in any header?